README.md

```diff
@@ -1,58 +1,5 @@
-<!-- START_METADATA
----
-title: Introduction to the Checkout API
-sidebar_label: Introduction
-sidebar_position: 1
-hide_table_of_contents: true
-description: Use the Checkout API to deliver a complete checkout experience, handling payment processing, shipping and gathering user information.
-pagination_next: null
-pagination_prev: null
----
-END_METADATA -->
-
-# Checkout API
-
-![Vipps](./images/vipps.png) *Available for Vipps now.*
-
-![MobilePay](./images/mp.png) *Available for MobilePay in selected markets at the [Vipps MobilePay joint platform launch](https://www.vippsmobilepay.com/about).*
-
-<!-- START_COMMENT -->
 ℹ️ Please use the website:
 [Vipps MobilePay Technical Documentation](https://developer.vippsmobilepay.com/docs/APIs/checkout-api>).
```

Comment:
This link does not work. Should it (be updated)?

Reply:
Oops! Thank you!! fixed now.